frontend/src/components/SystemModelsTable.tsx

```diff
@@ -20,21 +20,32 @@
 
 import React from "react";
 import { FormattedMessage } from "react-intl";
+import { graphql, useFragment } from "react-relay/hooks";
 
 import Table, { createColumnHelper } from "components/Table";
 import { Link, Route } from "Navigation";
+import type {
+  SystemModelsTable_SystemModelsFragment$key,
+  SystemModelsTable_SystemModelsFragment$data,
+} from "../api/__generated__/SystemModelsTable_SystemModelsFragment.graphql";
+// We use graphql fields below in columns configuration
```

Comment:
Let's divide block of imports with new line.
```suggestion
} from "../api/__generated__/SystemModelsTable_SystemModelsFragment.graphql";

// We use graphql fields below in columns configuration
```